Content.Server/GameTicking/Rules/Components/RevolutionaryRuleComponent.cs

```diff
@@ -59,6 +59,30 @@ public sealed partial class RevolutionaryRuleComponent : Component
     [DataField, ViewVariables(VVAccess.ReadWrite)]
     public int PlayersPerHeadRev = 15;
 
+    /// <summary>
+    /// Whether or not Head Revs have been selected yet or not.
+    /// </summary>
+    [DataField]
+    public bool HeadRevsChosen;
+
+    /// <summary>
+    /// When Revs will start during a shift.
+    /// </summary>
+    [DataField(customTypeSerializer: typeof(TimeOffsetSerializer)), ViewVariables(VVAccess.ReadWrite)]
+    public TimeSpan? StartTime;
```

Comment:
Needs offsetting on unpause.

Reply:
I'm confused, isn't that what timeoffsetserializer is for or am I mistaken?

Reply:
The serializer is for applying it when the component is read or copied, but you also need to add a subscription for EntityUnpausedEvent that increments the start time by the time that the rule entity was paused for.
Find-in-files `EntityUnpausedEvent>` should give you some examples to look at, such as the one in `AnomalySystem.Generator.cs`